corehq/apps/ota/tests/test_case_restore.py

```diff
@@ -103,6 +104,43 @@ def test_locations_in_restore(self):
         self.assertXmlPartialEqual(locations_content, response_content,
                                    '{http://openrosa.org/http/response}fixture[@id="locations"]')
 
+    @flag_enabled('ADD_LIMITED_FIXTURES_TO_CASE_RESTORE')
+    def test_lookup_table_in_restore(self):
+        case_id = self.dad.case_id
+
+        domain_obj = create_domain(self.domain)
+        user = WebUser.create(self.domain, 'test-user', 'passmein', created_by=None, created_via=None)
```

Comment:
I know you just copied this from the other test but it should not be necessary to create these.

Reply:
Fixed here: 55eac71f2db9c8104fa7a7a23afd4f011a89f35e
Thank you!